### PR TITLE
Reduce the analytics that are collected in Gradio

### PR DESCRIPTION
**Pull request by abidlabs**
Remove Google Analytics from Gradio apps and document the analytics that we do track from developers.

### CODE REVIEW

File: client/python/test/test_client.py [hunk 1]
```diff
@@ -324,6 +324,7 @@ def test_upload_and_download_with_auth(self):
         with open(output) as f:
             assert f.read() == "Hello file!"
 
+    @pytest.mark.flaky
```
Comment (abidlabs) [inline client/python/test/test_client.py:327]: unrelated but this is a flaky test and I'm not sure why

**Review by freddyaboulton — APPROVED** [review]

File: guides/01_getting-started/03_sharing-your-app.md [hunk 2]
```diff
@@ -464,3 +465,17 @@ demo.launch(max_file_size=5 * gr.FileSize.MB)
 ```
 
 Please make sure you are running the latest version of `gradio` for these security settings to apply.
+
```
Comment (aliabid94) [inline guides/01_getting-started/03_sharing-your-app.md:468]: Personally think this is way too much content about analytics in the getting started guide @abidlabs

**Review by julien-c — COMMENTED** [review]
nice!

File: guides/01_getting-started/03_sharing-your-app.md (hunk 2 above)
Comment (abidlabs) [reply guides/01_getting-started/03_sharing-your-app.md]: This guide is no longer in the getting started section :)

File: guides/03_additional-features/07_sharing-your-app.md [hunk 3]
```diff
@@ -464,3 +465,17 @@ demo.launch(max_file_size=5 * gr.FileSize.MB)
...
 Please make sure you are running the latest version of `gradio` for these security settings to apply.
+
+## Analytics
+
+By default, Gradio collects certain analytics to help us better understand the usage of the `gradio` library. This includes the following information:
+
+* What environment the Gradio app is running on (e.g. Colab Notebook, Hugging Face Spaces)
+* What input/output components are being used in the Gradio app
+* Whether the Gradio app is utilizing certain advanced features, such as `auth` or `show_error` 
+* The IP address which is used solely to measure the number of unique developers using Gradio
```
[the remark this answers is not in the document]
Comment (abidlabs) [reply guides/03_additional-features/07_sharing-your-app.md:476]: Ah yeah let me fix in another pr

**Pull request merged**